### DISCUSSION TITLE
Automatically watch for spam.

### DISCUSSION THREAD

**Discussion by TechStudent10** [ThioJoe/YT-Spammer-Purge#82] [Ideas]
Alright so here's how it works:
1. You start the program so that it watches for spam.
2. You let it run for 12 - 24 hours. Every few hours, it checks for spam comments. If found, they are written to a temporary file.
3. Once ready, you stop watching for spam. It gives you the list of spam comments. If it looks all good, it'll go ahead and delete them. If it is not okay, you can specify which comments to leave alone, and which to delete.

**Comment by KendallDoesCoding** [comment]
> Alright so here's how it works:
> 
> 1. You start the program so that it watches for spam.
> 2. You let it run for 12 - 24 hours. Every few hours, it checks for spam comments. If found, they are written to a temporary file.
> 3. Once ready, you stop watching for spam. It gives you the list of spam comments. If it looks all good, it'll go ahead and delete them. If it is not okay, you can specify which comments to leave alone, and which to delete.

Well, that's a good idea. But ideally, it should give a setting when first auto watching for spam, so that it doesn't slow down your pc (and internet, idk if it uses internet), and there can be a setting in the program too that if you type a certain command, it will auto start checking for spam from that day.
